### PR TITLE
Cleanup `configlint.py` into integration tool.

### LINKED ISSUE #30: Cleanup `configlint.py` into integration tool.
The functionality in `tests/configlint.py` may be useful to users that are trying to establish a set of packages/versions. We should cleanup `configlint.py` and turn it into an externalized tool.

### PR DESCRIPTION
**Pull request by bethune-bryant**
Resolves #30

### CODE REVIEW

File: validation/validate_env.py [hunk 1]
```diff
@@ -7,9 +7,7 @@
 US Government Users Restricted Rights - Use, duplication or
 disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 *****************************************************************
-"""
 
-"""
 *******************************************************************************
 Script: envlint.py
```
Comment (bnemanich) [inline validation/validate_env.py:12]: These comments need to be modified for the new script name and arguments.

File: tests/validate_config_test.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
+# *****************************************************************
```
Comment (bnemanich) [inline tests/validate_config_test.py:1]: It seems like some of this is duplicated in the build_env tests. Should some of it be moved to a common file?

File: open-ce/docker_build.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 HOME_PATH = "/home/builder"
 
 REPO_NAME = "open-ce"
-IMAGE_NAME = "open-ce-builder"
+IMAGE_NAME = "open-ce-open-ce"
```
Comment (bnemanich) [inline open-ce/docker_build.py:18]: Maybe this should still be builder?

Comment (bethune-bryant) [reply open-ce/docker_build.py:18]: Thanks, I tried to only replace the occurrences of builder that needed to be replaced, but I must have missed this one.

**Review by bnemanich — APPROVED** [review]

**Pull request merged**